src/java.base/share/classes/java/security/spec/MGF1ParameterSpec.java

```diff
@@ -96,12 +96,16 @@ public class MGF1ParameterSpec implements AlgorithmParameterSpec {
 
     /**
      * The MGF1ParameterSpec which uses SHA-512/224 message digest
```

Comment:
Feel free to cover this as a separate issue, but I think these constant descriptions should end with a period. Also, I would say "... uses _a_ SHA-512/224 message digest." (change in italics). There is also some inconsistency in the constants, some put the digest algorithm in double-quotes while others don't - it would be good to be consistent.

Reply:
Updated for consistency.